Source/Core/Core/Analytics.cpp

```diff
@@ -116,6 +118,61 @@ void DolphinAnalytics::ReportGameStart()
   Send(builder);
 }
 
+void DolphinAnalytics::ReportUnknownIOSDevice(const std::string& path)
+{
+  Common::AnalyticsReportBuilder builder = MakeBuilderWithGameID();
+  builder.AddData("type", "ios-unknown-device");
+  builder.AddData("ios-version", IOS::HLE::GetVersion());
+  builder.AddData("ios-path", path);
+  Send(builder);
+}
+
+void DolphinAnalytics::ReportUnknownIOSRequest(const std::string& path,
```

Comment:
One thing design-wise I'm not completely convinced about: intuitively, I think these report building functions should be defined near the IOS code, not near the analytics code. This is because otherwise we might end up growing too many dependencies from analytics to the rest of the codebase. I'd rather have the dependencies be in the other direction: random modules using analytics -> analytics.

But again: I'm not set on any of the two, I'd just like to make sure you considered the alternative.